lms/static/scripts/frontend_apps/components/EmailMentionsPreferences.tsx

```diff
@@ -0,0 +1,42 @@
+import { Checkbox } from '@hypothesis/frontend-shared';
+import classnames from 'classnames';
+
+export type EmailMentionsPreferencesProps = {
+  subscribed: boolean;
+};
+
+export default function EmailMentionsPreferences({
+  subscribed,
+}: EmailMentionsPreferencesProps) {
+  return (
+    <div className="flex flex-col gap-y-3">
+      <h2 className="text-lg/4 text-brand">Mentions</h2>
+      <p>
+        Receive email notifications when you are mentioned in an annotation.
+      </p>
+      <div className="px-4">
+        <span
+          className={classnames(
+            // The checked icon sets fill from the text color
+            'text-grey-6',
+          )}
```

Comment:
The styles on this `span` here could be folded into the `div` above, though see the comment about styling the checkbox icon in `EmailDigestPreferences`.